app/Exceptions/CDashXMLValidationException.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Exceptions;
+
+use Exception;
+use RuntimeException;
+
+class CDashXMLValidationException extends RuntimeException
```

Comment:
This could probably be renamed to `CDashXMLValidationException` since the namespace implies that it's part of CDash.

Reply:
@williamjallen. I assume that you mean `XMLValidationException`, dropping the unneeded `CDash`?

Reply:
Yep, my bad.  I copied the name and forgot to edit it 🤦‍♂️.